diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter2/KdocMethodsFixTest.kt

```diff
@@ -40,3 +35,10 @@ class KdocMethodsFixTest : FixTestBase("test/paragraph2/kdoc/package/src/main/ko
         fixAndCompare("KdocMethodsFullExpected.kt", "KdocMethodsFullTested.kt")
     }
 }
+
+class KdocMethodsFixTest : FixTestBase("test/paragraph2/kdoc/methods", KdocMethods()) {
+    @Test
+    fun `KDoc template should be placed before modifiers`() {
```

Comment:
This test if fixed in master branch, so this fix can be reverted here.

Reply:
reverted